app/assets/javascripts/host_edit.js

```diff
@@ -187,6 +183,15 @@ function submit_with_all_params() {
   return false;
 }
 
+function page_title_by_resource(resource) {
+  var capitalized_resource = resource[0].toUpperCase() + resource.slice(1),
+  page_state_title = capitalized_resource;
```

Comment:
I believe we can use only:
```suggestion
  var page_state_title = resource[0].toUpperCase() + resource.slice(1);
```
and drop the `capitalized_resource` variable alltogether, later inside the `if` we would:
```js
page_state_title = page_state_title + ' show';
```
What is perfectly fine statement and can be even shorened to `page_state_title += ' show'`